packages/atlas/src/views/studio/CrtDashboard/tabs/CrtRevenueTab.tsx

```diff
@@ -58,8 +58,8 @@ export const CrtRevenueTab = ({ token }: CrtRevenueTabProps) => {
           customNode={
             <RatioPreview
               ratios={[
-                [100 - token.annualCreatorReward, 'Holders'],
-                [token.annualCreatorReward, 'Channel'],
+                [100 - token.annualCreatorRewardPermill, 'Holders'],
```

Comment:
Now that this is called `annualCreatorRewardPermill` doesn't it mean that it should be divided by 10 ?

Reply:
It will be handled by https://github.com/Joystream/atlas/pull/5060